pkg/plugin/provider/certs.go

```diff
@@ -212,32 +212,58 @@ func RemoveCerts(namespace string) error {
 func UnImportCertificate(file certFile, namespace string) error {
 	dynamicDir := fmt.Sprintf("%s/%s", gmsaDirectory, namespace)
 
-	// get cert thumbprint using certutil. Thumbprints are equal to the sha1 hash of the certificate
+	logrus.Infof("Attempting to remove certificate %s", file.hostFile)
+
+	// get cert thumbprint using certutil. Thumbprints are equal to the sha1 hash of the certificate.
+	// Cert util will return a sha1 field in the following format
+	// Cert Hash(sha1): <someHash>
 	certUtilArgs := []string{"-Command",
-		fmt.Sprintf("$(certutil %s)", file.hostFile), "-like", "\"Cert Hash(sha1):*\""}
+		fmt.Sprintf("certutil %s", fmt.Sprintf("C:%s", file.hostFile)), "|", "Select-String", "-Pattern", "\"Cert Hash\\(sha1\\)\"",
+	}
+
+	cmd := exec.Command("powershell", certUtilArgs...)
+	logrus.Debugf("Crafted Command: %s", cmd.String())
 
-	o, err := exec.Command("powershell", certUtilArgs...).Output()
+	o, err := cmd.CombinedOutput()
 	if err != nil {
-		return fmt.Errorf("failed to obtain sha1 thumbPrint of cert in %s: %v", dynamicDir, err)
+		return fmt.Errorf("failed to obtain sha1 thumbPrint of cert in %s (%s): %v", dynamicDir, string(o), err)
```

Comment:
Can we move the `string(o)` into a `logrus.Info` output instead?

Reply:
Done